pkg/bench/queue_manager_info.go

```diff
@@ -0,0 +1,80 @@
+package bench
+
+import (
+	"net/url"
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus"
+	common_config "github.com/prometheus/common/config"
+	"github.com/prometheus/common/promlog"
+	"github.com/prometheus/prometheus/config"
+	"github.com/timescale/promscale/pkg/remote"
+)
+
+type qmInfo struct {
+	qm        *remote.QueueManager
+	samplesIn *remote.EwmaRate
+	highestTs *remote.MaxTimestamp
+}
+
+func (qmi *qmInfo) run() {
+	ticker := time.NewTicker(shardUpdateDuration)
+	defer ticker.Stop()
+	for range ticker.C {
+		qmi.samplesIn.Tick()
+	}
+}
+
+func getQM(conf *BenchConfig) (*qmInfo, error) {
+	url, err := url.Parse(conf.WriteEndpoint)
+	if err != nil {
+		return nil, err
+	}
+
+	rwConf := config.DefaultRemoteWriteConfig
```

Comment:
I havent read evreything, but Prometheus uses [this](https://github.com/prometheus/prometheus/blob/cdc71a1c2ac23d0ed2f7284c2a667fbfc1296414/config/config.go#L115). I wonder if we should use the same, unless some other reason.